src/components/Nav/nav.jsx

```diff
@@ -1,13 +1,20 @@
-import { Link } from "react-router-dom";
+import { NavLink, useNavigate } from "react-router-dom";
```

Comment:
Can you please explain the purpose of using NavLink instead of just Link? Did not quite understand its purpose and the difference between the two.

Reply:
Sure, so Link is used when you have to traverse through different routes. Navlink helps when you want to add some style attribute to it, for example if you want to show the route that's currently active which can be useful in nav bar. Though I haven't added any style yet, I intend to come back to this later. This might help in understanding the difference better - https://stackoverflow.com/questions/47338077/react-router-v4-navlink-vs-link-benefits

Reply:
For the login part, I've currently put the test credentials (Adarsh Balika) as the initial state. Hence, you'd just need to click on the login button and you're in!

Reply:
> Sure, so Link is used when you have to traverse through different routes. Navlink helps when you want to add some style attribute to it, for example if you want to show the route that's currently active which can be useful in nav bar. Though I haven't added any style yet, I intend to come back to this later. This might help in understanding the difference better - https://stackoverflow.com/questions/47338077/react-router-v4-navlink-vs-link-benefits

Okay, now I understand! Thanks for this explanation!